### ISSUE TITLE
Why is CountryCode.TW "Taiwan, Province of China"?

### ISSUE THREAD

**Issue by stickfigure** [TakahikoKawasaki/nv-i18n#81]
The explicit label "Province of China" seems unnecessarily political. If I print the country name text as-is in my app I'm at risk of making it look like my app has a political opinion. 

It's mildly annoying having to filter this bit of text out. How about we change the text to simply "Taiwan"?

**Comment by hshar7** [comment]
Yep.. just got a complaint about this from a user and we have to filter it out now

**Comment by stickfigure** [comment]
Not my preference, but I've forked this project and published a drop-in compatible library in maven central:

https://github.com/stickfigure/nv-i18n-safe

```
		<dependency>
			<groupId>com.voodoodyne</groupId>
			<artifactId>nv-i18n-safe</artifactId>
			<version>1.29</version>
		</dependency>
```

I also made a few other names human readable, like "South Korea" and "North Korea". Everything should be suitable for displaying in UIs now.

**Comment by marcwrobel** [comment]
@stickfigure, _Taiwan, Province of China_ is the name given by the International Organization for Standardization (ISO) for the TW code (https://www.iso.org/obp/ui/#iso:code:3166:TW).

**Comment by stickfigure** [comment]
@marcwrobel no matter what the ISO says, that name is politically unsuitable for use in UIs outside of China. The Koreas are also problematic - I doubt half the world's population could tell the difference between "Korea, Democratic People's Republic of" and "Korea, Republic of".

I need text I can display in UIs so folks can ship packages to those places. The ISO names for countries don't quite work. So it's either convince nv-i18n to change, make a fork, or figure out some sort of layer on top of nv-i18n that gives me access to friendly names. The fork got me up and running, but it's not my favorite choice.

If this is just a pedantic issue, can nv-i18n add a "friendly name" field as well as the ISO name? I would be happy to submit a PR for that.

**Comment by marcwrobel** [comment]
@stickfigure, I did not try to make a point, I just gave an answer to the question you asked in this issue's title : _Why is CountryCode.TW "Taiwan, Province of China"?_.

And I agree with you : the name provided by the ISO are not suitable for a UI. But I also don't think the goal of nv-i18n is to provide suitable names for any UI, mainly because such names are intrinsically political (e.g. Taiwan,, Palestine...), but also because many softwares need to be internationalized (nv-i18n does not provide such features - see also #19).
